internal/gcp/tags.go

```diff
@@ -178,3 +180,45 @@ func (m *tagsManager) GetProjectInfo(ctx context.Context, projectID string) (*re
 	m.cache.Set(cacheKey, project, tagCacheDuration)
 	return project, nil
 }
+
+func (m *tagsManager) DeleteValue(ctx context.Context, projectID string, key string, value string) error {
```

Comment:
Should this maybe also be named `DeleteValueIfUnused` like the other function for consistency?

Reply:
Sure makes sense, I will change the name